casestudies/build.sh

```diff
@@ -12,4 +12,4 @@ patch -p0 < ../diffs/mime.diff
 cd ..
 
 # patch hippy-c-cgi
-patch -p0 < diffs/hippycgi.diff
+#patch -p0 < diffs/hippycgi.diff
```

Comment:
This depends on how PyHyp is compiled (it used to be pyhyp, but now it seems to be hippy-c). So I'll have to update this once I'm sure which way it is. Guess I can check it now.